README.md

```diff
@@ -597,3 +598,9 @@ gitLabApi.getUserApi().createUser(userConfig, password, sendResetPasswordEmail);
 // Get a list of pages in project wiki
 List<WikiPage> wikiPages = gitLabApi.getWikisApi().getPages();
 ```
+
+#### TagsApi
+```java
+// Get a list of tags for the specified project ID
+List<Tag> tags = gitLabApi.getTagsApi().getTags(projectId);
+```

```

Comment:
This needs to be in alphabetical order, please move it to after the ```SystemHooksApi``` section.

Reply:
Ok understood.